src/tasks/mixin/battle_mixin.py

```
@@ -567,19 +567,27 @@ def auto_battle(self, no_battle: bool = False):
+                if self.is_battle_settlement():
                     self.log_info("检测到战斗结算状态，战斗完成")
-                else:
+                    return True
+
+                if self.in_combat(required_yellow=1):
+                    last_battle_time = None
+                    sleep_time = 0.1
+                    continue
+
+                if battle_elapsed >= 15:
                     self.log_info("战斗结束状态等待超时，继续后续结算检测")
```

Ruff (0.16.0)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)